backend/tests/test_course_build_processor.py

```diff
@@ -240,7 +288,7 @@ def test_non_existing_course_build(mocker):
     )
 
     sqs_stubber.activate()
-    mocker_map = {"sqs": sqs_client}
+    mocker_map = {"sqs": sqs_client, "s3": s3_client}
```

Comment:
Minor: I think on these tests you don't really need to pass a valid client. Something like the following should work too which would prevent us from having to define an (effectively unused) `s3_client` in them:

```python
    mocker_map = {"sqs": sqs_client, "s3": None}
```